lib/idx/flow/RegistrationFlowMonitor.ts

```diff
@@ -27,4 +27,8 @@ export class RegistrationFlowMonitor extends FlowMonitor {
 
     return super.isRemediatorCandidate(remediator, remediations, values);
   }
+
+  protected getKnownCyclicRemediators() {
```

Comment:
Looks like the loop detect logic start blocking us, I also saw issues in other scenarios. We probably can remove it sometime. (keep it in this PR)
I have a draft PR here: https://github.com/okta/okta-auth-js/pull/1032